test/com/facebook/buck/features/rust/RustLibraryIntegrationTest.java

```diff
@@ -96,7 +97,8 @@ public void rustLibraryCheckWarning() throws IOException {
                 "rust.rustc_check_flags=-Dwarnings --cfg \"feature=\\\"warning\\\"\"",
                 "//messenger:messenger#check")
             .getStderr(),
-        containsString("error: method is never used: `unused`"));
+        either(containsString("error: method is never used: `unused`"))
+            .or(containsString("error: associated function is never used: `unused`")));
```

Comment:
This file is unrelated right? Let's pull this out.

Reply:
Done. Thanks